src/foam/core/FOAMException.js

```diff
@@ -125,7 +128,7 @@ foam.CLASS({
       name: 'getTranslation',
       type: 'String',
       code: function() {
-        if ( ! this.translationService ) return msg;
```

Comment:
if curious - this came from https://github.com/kgrgreer/foam3/commit/5aad2045a9627b7736c5cecca4e8a214d3807bc6